modin/core/execution/ray/common/task_wrapper.py

```diff
@@ -39,10 +43,14 @@ def _deploy_ray_func(func, args):  # pragma: no cover
     ray.ObjectRef or list
         Ray identifier of the result being put to Plasma store.
     """
-    return func(**args)
+    logger = get_worker_logger(JOB_ID)
+    logger.info(f"START::PANDAS-API::deploy_ray_func.{func.__qualname__}")
```

Comment:
```suggestion
    logger.info(f"START::RAY-REMOTE::deploy_ray_func:{func.__qualname__}")
```
I suggest changing both "modin layer" to signify it's Ray and to change dot to colon to highlight that `func.__qualname__` is not a child of `deploy_ray_func` (however the latter is questionable as it could potentially break log parsing if it exists already)